### ISSUE TITLE
[Tensorflow 1 is deprecated] Support will be removed on August 1, 2022.

### ISSUE THREAD

**Issue by kngoc1810** [orpatashnik/StyleCLIP#98]
When running setup for **StyleCLIP_global**, this warning message shown. 
```
WARNING: Tensorflow 1 is deprecated, and support will be removed on August 1, 2022.
After that, `%tensorflow_version 1.x` will throw an error.

Your notebook should be updated to use Tensorflow 2.
See the guide at https://www.tensorflow.org/guide/migrate#migrate-from-tensorflow-1x-to-tensorflow-2.

TensorFlow 1.x selected.
```
Does this mean I will not able to run this model after  August 1, 2022? If yes, what can I do?

**Comment by zhumengzj** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，来信已收到，谢谢！

**Comment by betterze** [comment]
We add support for global direction with torch implementation in [here](https://colab.research.google.com/github/orpatashnik/StyleCLIP/blob/main/notebooks/StyleCLIP_global_torch.ipynb)
